src/helper.cc

```diff
@@ -869,9 +874,13 @@ helper::handleKilledServer(HelperServerBase *srv, bool &needsNewServers)
 
         if (childs.needNew() > 0) {
             srv->flags.shutdown = true;
-            needsNewServers = true;
+            debugs(80, DBG_IMPORTANT, "Starting new helpers");
```

Comment:
Let's not preserve this debugs() because helperOpenServers() already has (an arguably better) one, at the same debugging level.

```suggestion
```

BTW, we did not preserve a similar debugs() in helper_stateful_server::HelperServerClosed().

Reply:
Removed.